README.rst

```diff
@@ -11,6 +11,21 @@ About
 =====
 This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
+PyMongoExplain greatly simplifies the amount of effort needed to explain commands.
+For example, suppose we wanted to explain the following ``update_one``: ::
```

Comment:
Here and below, I think you can replace `: ::` with just `::`

Reply:
Done.